Link the lazer feature comparison from `Release stream` and `Lazer` articles (#8925)
ppy/osu-wiki: wiki/Client/Release_stream/en.md

| Stable | Current, most popular version; [feature-locked](https://en.wikipedia.org/wiki/Freeze_(software_engineering)); receives updates after Beta |
| Beta | Receives updates after Cutting Edge; feature-locked |
| Cutting Edge | Receives newest updates; feature-locked |

> | [Lazer](Lazer) | Rewritten game client; most active in development with [many differences](/wiki/Client/Release_stream/Lazer/Feature_comparison) from the stable version. |

Reviewer: ```suggestion
| [Lazer](Lazer) | Rewritten game client; most active in development with [many differences and new features](/wiki/Client/Release_stream/Lazer/Feature_comparison). |
```
